.github/workflows/release.yaml

```diff
@@ -0,0 +1,45 @@
+name: release
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+([0-9]+)'
+
+jobs:
+  docker-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Log in to the Container registry
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v5
+      # The latest tag refers to the latest released version
+      - name: Build and push Docker image to sceptreorg/sceptre:latest
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          push: true
+          tags: sceptreorg/sceptre:latest
+          labels: ${{ steps.meta.outputs.labels }}
+      - name: Build and push Docker image to sceptreorg/sceptre:${{ steps.meta.outputs.tags }}
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          push: true
+          tags: sceptreorg/sceptre:${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+  pypi-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Poetry
+        uses: snok/install-poetry@v1
+      - name: Install dependencies
+        run: poetry install --no-interaction --all-extras
```

Comment:
This shouldn't be strictly necessary for a release should it? The `publish` would be all you really need to do.

Reply:
hmm, maybe?  i'll test and see.